nvm.sh

```diff
@@ -1545,6 +1564,13 @@ nvm() {
       if [ "_$1" = "_-s" ]; then
         nobinary=1
         shift
+        if [ "_$1" = "_-j" ]; then
+          shift
```

Comment:
`-j` has a following parameter, so it has two shift, `-s` doesn't, isn't it correct?